templates/mirrors/mirror_submit.html

```diff
@@ -0,0 +1,34 @@
+{% extends "base.html" %}
+{% load package_extras %}
+{% block title %}Arch Linux - Pacman Mirrorlist Generator{% endblock %}
+
+{% block content %}
+<div id="mirrorlist-gen" class="box">
+
+    <h2>Mirror Request</h2>
+
+    <p>This page is meant as a replacement of the old way of registring as a Tier 1 or Tier 2 mirror. Previously this was done through <a href="https://bugs.archlinux.org/index.php?string=&project=1&search_name=&type%5B%5D=&sev%5B%5D=&pri%5B%5D=&due%5B%5D=&reported%5B%5D=&cat%5B%5D=43&status%5B%5D=open&percent%5B%5D=&opened=&dev=&closed=&duedatefrom=&duedateto=&changedfrom=&changedto=&openedfrom=&openedto=&closedfrom=&closedto=&do=index">https://bugs.archlinux.org</a> and would require manual intervention from the mirror maintainer(s). This process is now semi-automated and various checks against your mirror will be performed when submitting via the below form.</p>
+
+    <h3>Available mirrors</h3>
+
+    <p>Below are direct links to the two different tiers you will need to be acquainted with.<p>
+
+    <ul>
+        <li><a href="/mirrors/tier/1/">Tier 1 mirrors</a></li>
+        <li><a href="/mirrors/tier/2/">Tier 2 mirrors</a></li>
+    </ul>
+
+    <h3>Mirror information</h3>
+
+    <p>Before you can submit a <b>Tier 1</b> request the mirror in question must first be a registered <b>Tier 2</b> for a certain amount of time with proven reliablity. Once the submitted information is verified the mirror will be visible under the appropriate tier list above. This process usually takes 5 minutes.</p>
```

Comment:
The way I would interpert this is that my mirror will be accepted within 5 minutes after submission that sounds a bit optimisitic :)

Reply:
We can tweak it. But there's not much information in general what's the guiding values regarding any mirror related tasks.
5 min was a ballpark figure after having administrated the mirror list for a few days now. It usually takes less for the mirror "Completion" rate to be registered.
The idea was to automate the `Inactive` -> `Active` status once the first sync/check is performed.

For now the process will remain manual, and yea it will take quite a bit longer than 5 minutes hehe.

Just want to remind that this is a draft, and an attempt to convert the [manual process](https://wiki.archlinux.org/title/DeveloperWiki:NewMirrors) into at the very least semi-automated one ^^